kafka_helpers/OperationEnum.go

```diff
@@ -0,0 +1,26 @@
+package kafka_helpers
+
+type OperationEnum int
+
+const (
+	Created OperationEnum = iota
+	Updated
+	Deleted
+)
+
+func OperationEnumString(e OperationEnum) string {
```

Comment:
FYI, there is a way you can do this in Go more efficiently:
const (
    Foo MyEnum = iota + 1
    Bar
)

func (me MyEnum) String() string {
    return [...]string{"", "Foo", "Bar"}[me]
}